waveform-django/waveforms/models.py

```diff
@@ -1,11 +1,12 @@
 from django.db import models
 
-
 class User(models.Model):
     username = models.CharField(max_length=150, unique=True, blank=False,
         default='')
     join_date = models.DateTimeField(auto_now_add=True)
     is_admin = models.BooleanField(default=False)
+    requested_annotations = models.BooleanField(default=False)
+    when_requested = models.DateTimeField(auto_now_add=True)
```

Comment:
Does `auto_now_add=True` work for this use case? That stuff always confuses me

Reply:
While using `auto_now_add=True` worked as intended, I took a closer look at the documentation for it and the documentation states:
> Note that the current date is always used; it’s not just a default value that you can override. So even if you set a value for this field when creating the object, it will be ignored.

So to err on the side of caution, I'll change it to the recommended ` django.utils.timezone.now()`